tests/CMakeLists.txt

```diff
@@ -47,19 +62,28 @@ MACRO(ROBOPTIM_CORE_TEST NAME)
   ADD_TEST(NAME ${NAME}
     COMMAND ${CHECK_PREFIX} ${CMAKE_CURRENT_BINARY_DIR}/${NAME})
 
+  #Adding a project configuration file (for MSVC only)
+  GENERATE_MSVC_DOT_USER_FILE(${NAME})
+
   TARGET_LINK_LIBRARIES(${NAME} roboptim-core)
-  PKG_CONFIG_USE_DEPENDENCY(${NAME} eigen3)
   PKG_CONFIG_USE_DEPENDENCY(${NAME} liblog4cxx)
+  IF(NOT LTDL_FOUND)
+    TARGET_LINK_LIBRARIES(${NAME} ltdl)
+  ELSE()
+    PKG_CONFIG_USE_DEPENDENCY(${NAME} ltdl)
+  ENDIF(NOT LTDL_FOUND)
 
   # Link against Boost.
   TARGET_LINK_LIBRARIES(${NAME} ${Boost_LIBRARIES})
-
+  
   # Make sure the plugins will be found.
   # Note: this does not append the environment variables!
+  IF(NOT WIN32)
   SET_PROPERTY(
     TEST ${NAME} PROPERTY ENVIRONMENT
-    "LTDL_LIBRARY_PATH=${CMAKE_BINARY_DIR}/src:$ENV{LTDL_LIBRARY_PATH} "
-    "LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/src:$ENV{LD_LIBRARY_PATH}")
+    "LTDL_LIBRARY_PATH=${CMAKE_BINARY_DIR}/src/${CMAKE_BUILD_TYPE}:$ENV{LTDL_LIBRARY_PATH} "
+    "LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/src/${CMAKE_BUILD_TYPE}:$ENV{LD_LIBRARY_PATH}")
+  ENDIF(NOT WIN32)
```

Comment:
This changes the location of built `.so` to `src/${CMAKE_BUILD_TYPE}` for Linux, yet this did not change, right?